builder-support/debian/authoritative/debian-buster/config/ixfrdist.yml

```diff
@@ -0,0 +1,102 @@
+# ixfrdist example configuration file.
+# Please see man 5 ixfrdist.yml for details.
+
+# The directory where the domain data is stored. When unset, the current
+# working directory is used. Note that this directory must be writable for the
+# user or group ixfrdist runs as.
+#
+work-dir: '/var/lib/ixfrdist'
+
+# User to drop privileges to once all listen-sockets are bound. May be either
+# a username or numerical ID.
+#
+uid: _ixfrdist
+
+# Group to drop privileges to once all listen-sockets are bound. May be either
+# a username or numerical ID.
+#
+gid: _ixfrdist
+
+# Listen addresses. ixfrdist will listen on both UDP and TCP.
+# When no port is specified, 53 is used. When specifying ports for IPv6, use the
+# "bracket" notation:
+#
+#    listen:
+#      - '127.0.0.1'
+#      - '::1'
+#      - '192.0.2.3:5300'
+#      - '[2001:DB8:1234::334]:5353'
+#
+# By default, ixfrdist listens on local loopback addresses.
+#
+#listen:
+#  - '127.0.0.1'
+#  - '::1'
+
+# Netmasks or IP addresses of hosts that are allowed to query ixfrdist. Hosts
+# do not need a netmask:
+#
+#    acl:
+#      - '127.0.0.0/8'
+#      - '::1'
+#      - '192.0.2.55'
+#      - '2001:DB8:ABCD::/48'
+#
+# By default, only loopback addresses are allowed.
+#
+#acl:
+#  - '127.0.0.0/8'
+#  - '::1'
+
+# Maximum number of records allowed in a single zone. ixfrdist will abort the
+# zone transfer from the master when more than this number of records have been
+# received. A value of 0 means unlimited
+#
+#axfr-max-records: 0
+
+# Timeout in seconds an AXFR transaction requested by ixfrdist may take.
+# Increase this when the network to the authoritative servers is slow or the
+# domains are very large and you experience timeouts.
+#
+#axfr-timeout: 20
+
+# Time in seconds between retries of the SOA query for a zone we have never
+# transferred.
+#
+#failed-soa-retry: 30
+
+# Whether record compression should be enabled, leading to smaller answers
+# at the cost of an increased CPU and memory usage.
+#
+#compress: false
```

Comment:
they are however, completely different file formats, and are written by different pieces of software. this is not introduced in this PR, so please fix this elsewhere first, otherwise it will always be reintroduced.